easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.0-GCC-8.2.0-2.31.1-hpcx.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '4.0.0'
+versionsuffix = '-hpcx'
+
+homepage = 'http://www.open-mpi.org/'
+description = """Mellanox flavored Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+sources = [{
+    'filename': 'openmpi-4.0.x.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/open-mpi',
+        'repo_name': 'ompi',
+        'commit': '03cf3e4',
```

Comment:
And repeating what i said in the other comment. Do not set the version of this OpenMPI build to 4.0.0 if it is not using the official 4.0.0 release from OpenMPI!